Dockerfile

```diff
@@ -1,14 +1,28 @@
-FROM node:8
+FROM node:8-alpine
```

Comment:
As I mentioned, I don't think it's the best to switch to alpine now

Reply:
Are there any dependencies on the operating system? Alpine is a lot smaller than the debian.